po/ja.po

```diff
@@ -0,0 +1,561 @@
+msgstr "クリップボードに名前・パスワードの一組を追加する"
+
+#. Translators: help for gpaste rename-password <old name> <new name>
+#: src/client/gpaste-client.c:169
+msgid "old name"
+msgstr "旧名"
+
+#: src/client/gpaste-client.c:169
+msgid "new name"
+msgstr "新しい名前"
+
+#: src/client/gpaste-client.c:169
+msgid "rename the password"
+msgstr "パスワードの名前を変更する"
+
+#. Translators: help for gpaste get <number>
+#. Translators: help for gpaste merge <number> … <number>
+#. Translators: help for gpaste set-password <number> <name>
+#. Translators: help for gpaste delete <number>
+#: src/client/gpaste-client.c:171 src/client/gpaste-client.c:177
+#: src/client/gpaste-client.c:179 src/client/gpaste-client.c:181
+msgid "number"
+msgstr "数字"
+
+#: src/client/gpaste-client.c:171
+msgid "get the <number>th item from the history"
+msgstr "履歴から<number>番目の項目を取得する"
+
+#. Translators: help for gpaste select <uuid>
+#: src/client/gpaste-client.c:173
+msgid "set the item matching this id from the history to the clipboard"
+msgstr "この ID に一致する項目を履歴からクリップボードに設定する"
+
+#. Translators: help for gpaste replace <number> <contents>
+#: src/client/gpaste-client.c:175
+msgid "contents"
+msgstr "内容"
+
+#: src/client/gpaste-client.c:175
+msgid ""
+"replace the contents of the item matching <uuid> from the history with the "
+"provided one"
+msgstr "履歴の <uuid> に一致する項目の内容を提供されたもので置き換える"
+
+#: src/client/gpaste-client.c:177
+msgid ""
+"merge the <number>th items from the history and add put the result in the "
+"clipboard"
+msgstr "履歴から<number>番目の項目をマージして、結果をクリップボードに追加する"
+
+#: src/client/gpaste-client.c:179
+msgid "set the <number>th item from the history as a password named <name>"
+msgstr "履歴から<number>番目の項目を<name>という名前のパスワードとして設定する"
+
+#: src/client/gpaste-client.c:181
+msgid "delete <number>th item of the history"
+msgstr "履歴から<number>番目の項目を削除する"
+
+#: src/client/gpaste-client.c:183
+msgid "delete the password <name> from the history"
+msgstr "履歴からパスワード<name>を削除する"
+
+#. Translators: help for gpaste file <path>
+#: src/client/gpaste-client.c:185
+msgid "path"
+msgstr "パス"
+
+#: src/client/gpaste-client.c:185
+msgid "put the content of the file at <path> into the clipboard"
+msgstr "<path>にあるファイルの内容をクリップボードに入れる"
+
+#. Translators: help for whatever | gpaste
+#: src/client/gpaste-client.c:187
+msgid "whatever"
+msgstr "プログラム"
+
+#: src/client/gpaste-client.c:187
+msgid "set the output of whatever to clipboard"
+msgstr "プログラムの出力をクリップボードに設定する"
+
+#. Translators: help for gpaste empty
+#: src/client/gpaste-client.c:189
+msgid "empty the history"
+msgstr "履歴を空にする"
+
+#. Translators: help for gpaste start
+#: src/client/gpaste-client.c:191
+msgid "start tracking clipboard changes"
+msgstr "クリップボードの変更の追跡を開始する"
+
+#. Translators: help for gpaste stop
+#: src/client/gpaste-client.c:193
+msgid "stop tracking clipboard changes"
+msgstr "クリップボードの変更の追跡を停止する"
+
+#. Translators: help for gpaste quit
+#: src/client/gpaste-client.c:195
+msgid "alias for stop"
+msgstr "停止の別名"
```

Comment:
```suggestion
msgstr "gpaste stop と同じ"
```

It's an alias for `gpaste stop` so `gpaste stop` and `gpaste quit` should have the same behavior. So we can translate it the same as the translation of the description of `gpaste stop` or something like this.

Reply:
I see. Thanks for the explanation.